isomerpages/isomercms-frontend
# hotfix/v0.24.1

## Problem

Feature tour had a UI bug where it would open on top of the model. Eg.

![Screenshot 2023-04-27 at 9 38 05 PM](https://user-images.githubusercontent.com/42832651/234935276-df4eb2b1-cb0c-4549-8ad9-dcaf7f32f73f.png)

## Solution

Only render feature tour iff review request model is closed. 

## Tests 

1. Login via email + clear local storage for `dashboard-identity-feature-tour-v1`
2. go to a website with no approved review
3. click on 'request for review'
4. note that the red point thing that triggers the feature tour dont come up
